src/nupic/os/Timer.cpp

```diff
@@ -131,76 +136,80 @@ static inline nupic::UInt64 getTicksPerSec()
 
 #endif
 
-Timer::Timer(bool startme)  
+namespace nupic
```

Comment:
Everything below here is the same code but indented inside the namespace. This had to be done since I removed `using namespace nupic` at the top (which caused collision of the `Handle` type in CoreServices with `nupic::Handle`).